server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -350,6 +359,10 @@ protected void doMerge(final ObjectMapper mergeWith) {
         }
     }
 
+    protected long columnOID() {
```

Comment:
same

Reply:
it's needed it's not a sub class of FieldMapper